install/README.md

```diff
@@ -1,3 +1,48 @@
 # Install
 
 Resources for installing Gitpod
+
+## Replicated/KOTS
+
+Gitpod has teamed up with Replicated to simplify the installation of Gitpod for self-hosted users. This provides an installation interface, simplified license delivery for both community and paying users, easier upgrades and improved
```

Comment:
Looks like the sentence got cut-off in the middle 🤔

Reply:
Lucky I'm not a native English speaker...

![Alt Text](https://media.giphy.com/media/jUwpNzg9IcyrK/giphy.gif)